src/components/ebay-button/button.stories.js

```diff
@@ -91,6 +91,17 @@ export default {
                 },
             },
         },
+        variant: {
+            options: ['destructive'],
```

Comment:
This is close to what we were thinking.
We should have `options: ['destructive', 'standard']`

Reply:
Gotcha! Yea, being explicit here makes more sense. Ok, adding...